Mathlib/FieldTheory/PurelyInseparable.lean

```diff
@@ -359,6 +359,42 @@ alias AlgEquiv.perfectClosure := perfectClosure.algEquivOfAlgEquiv
 
 end map
 
+-- TODO: move to suitable location
+instance IntermediateField.charZero [CharZero F] (L : IntermediateField F E) :
+    CharZero L := charZero_of_injective_algebraMap (algebraMap F _).injective
```

Comment:
I don't know where to put these results into. The `IntermediateField` file doesn't import things about characteristic.

Reply:
Have you tried `find_home`? You can at worst create a new file.

Reply:
`find_home` suggests `Mathlib` if I remembered correctly. Is it a good idea to create a new file `Algebra/CharP/IntermediateField`?

Reply:
I don't see any reason to not do so.